app/Http/routes.php

```diff
@@ -56,3 +56,5 @@
     // signups
     Route::post('signups', 'Api\SignupsController@store');
 });
+
+Route::get('api/v2/activity', 'Api\ActivityController@index');
```

Comment:
@chloealee ah ya. I wonder if we should specify that a route needs the `api` middleware in the controller instead of defining it on the group so that all these can be grouped together.